### PR TITLE
mobile: adjusted CSS & HTML to optimize log in screen

### PR DESCRIPTION
**Pull request by garrett**
Fixing log in screen for #8035.

(Screenshots included in the issue)

### CODE REVIEW

**Comment by andreasn** [comment]
Overlapping text between 638px and 768px

screenshot from 2017-11-14 14-25-50

**Comment by andreasn** [comment]
The above happens when I use my phone in landscape mode.

File: src/ws/login.css [hunk 1]
```diff
@@ -605,3 +605,47 @@ body {
...
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        text-align: center;
+    }
+
+    .login-pf #brand {
+        position: static;
+        font-size: inherit;
```
Comment (andreasn) [inline src/ws/login.css]: This seems to make it very small compared to before. Is that on purpose?

**Comment by martinpitt** [comment]
I tested this on my Sony Xperia X compact (with Opera), and this is indeed nicer! Now the OSK does not cover the user/password fields any more:

screenshot_20171114-143649
screenshot_20171114-143710

It appears the extra attribute to not auto-capitalize the user name does not work in Opera, but that's not a regression. I also confirm that on my desktop (Firefox 57) the login page remains unchanged.

**Comment by martinpitt** [comment]
Not merging yet, as @andreasn found some issues. I'm fine with them becoming a follow-up fix, though.

**Comment by garrett** [comment]
@andreasn: I guess I need to adjust the breakpoint value. 640px probably should be about right. I think then it'd trigger on portrait on phones.

I'll make the change and push.

**Comment by garrett** [comment]
I've adjusted the breakpoint so phones in portrait get the modified page. Phones in landscape (typically) do not, nor do tablets. (Some of these _will_ get PatternFly's mobile mods, but not my mods to avoid the keyboard on top of PatternFly's.)

Here are screenshots at different widths from Firefox's dev device emulator:

# Phones

## iPhone 6+

i6plus


## Nexus 6
n6

# Tablets

## Nexus 7 (portrait)
n7-tablet

## iPad Mini 2 (portrait)
ipadmini2

## iPad Mini 2 (landscape)

ipadmini2-landscape

**Comment by garrett** [comment]
Actual screenshot from my Nexus 5 (which has a "small" screen these days). It's using the keyboard at a normal height.

screenshot_20171115-172521

**Pull request closed without merging**